### ISSUE TITLE
Web Component Identifier: Develop Alpha 1

### ISSUE THREAD

**Issue by heymatthenry** [uswds/uswds-next#10]
One of the first components we'll recreate as a Web Component is the `usa-identifier` (uswds/uswds#5706).

## Necessary Tasks

These are the features that should constitute MVP for the pre-release version, since they will help us get our arms around the architectural choices we'll have to make in developing further components.

- [ ] Add `vitest` unit tests for component (WIP `vitest` setup is currently happening in #8)
- [ ] Add i18n support
- [ ] Ensure feature parity with existing banner component
- [ ] Ensure consistency of visual design with existing banner component
- [ ] Evaluate component's accessibility shortcomings and document them
- [ ] Add a Storybook story for the component (Depends on #6)

## Stretch Goals

All of these will need to happen at some point, but don't necessarily have to happen for the pre-release version

- [ ] Figure out how to package the component by itself for distribution (Depends on #5)
- [ ] Remediate any accessibility bugs

**Comment by brunerae** [comment]
Close in favor of https://github.com/uswds/uswds-next/issues/10
